marimo/_server/export/exporter.py

```diff
@@ -115,7 +115,7 @@ def export_as_script(
         ]
         code = (
             f'\n__generated_with = "{__version__}"\n\n'
-            + "\n\n# ---\n\n".join(codes)
```

Comment:
Thanks for implementing this. I'd add `# %%` before the first cell also.

Reply:
I updated to be right before each cell, and included on the first cell